fixtures/generate.py

```diff
@@ -0,0 +1,75 @@
+# For instructions on using this script, please see the README.
+
+from tuf import repository_tool as rt
+import os
+import shutil
+
+from os import path
+from unittest import mock
+
+# This file largely derives from the TUF tutorial:
+# https://github.com/theupdateframework/tuf/blob/develop/docs/TUTORIAL.md
+
+def import_keypair(name):
+    dir = path.join(os.getcwd(), 'keys')
+    private_path = path.join(dir, name)
+    public_path = private_path + '.pub'
+
+    # Load the keys into TUF.
+    public = rt.import_ed25519_publickey_from_file(public_path)
+    private = rt.import_ed25519_privatekey_from_file(private_path, password='pw')
+
+    return (public, private)
+
+def add_target(repository, filename, signing_target=None):
```

Comment:
Do we actually need this function? We only call it once and `signing_target` is always sent.

Reply:
We call it twice:

```
    add_target(repository, 'packages.json')
    add_target(repository, 'p2/drupal/core.json')
```

It's highly likely we'll call it even more than twice, as we add more target files to the fixture (which is something I fully expect to happen). So I'm leaving this as-is for now.

Reply:
do we actually need the logic for making `signing_target` optional though?

Reply:
Well, truth be told, I don't know. This is one of the places where I was like "TUF works in mysterious ways", and just left it alone. I'm open to removing it, but it seems possible that we might want to use it at some point in the future. So I figured I'd leave it there.

Reply:
I think we should remove because we need the if/else then. We may never use that code path. 
so `def add_target(repository, filename, signing_target):`